hvplot/converter.py

```diff
@@ -90,6 +90,11 @@
 )
 from .utilities import hvplot_extension
 
+try:
```

Comment:
I would really prefer if we avoided importing geoviews here at the top-level:
- It's not great for performance reasons
- It comes with side-effects I'd rather avoid as much as possible